sjb/generate.py

```diff
@@ -108,7 +108,7 @@ def load_configuration(config_path):
     # now, the job can define actions to take
     for action in job_config.get("actions", []):
         if action["type"] == "script":
-            actions.append(ScriptAction(action.get("repository", None), action["script"], action.get("title", None), action.get("timeout", 7200)))
+            actions.append(ScriptAction(action.get("repository", None), action["script"], action.get("title", None), action.get("timeout", 14400)))
```

Comment:
I think the pattern (which you could argue is bad) was to default to `None` here and only have the one default -- in the action definition